usage/rest-server/src/main/java/brooklyn/rest/resources/ServerResource.java

```diff
@@ -191,6 +232,15 @@ public void run() {
                 }
             }
 
+            private boolean hasStoppingApps(ManagementContext mgmt) {
```

Comment:
All apps are stopping by now, but you are right the suggested name will better reflect the code in it.